src/Composer/Installers/EzPlatformInstaller.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Composer\Installers;
+
+class EzPlatformInstaller extends BaseInstaller
+{
+    protected $locations = array(
+        'meta-assets' => 'web/assets/ezplatform/',
+        'assets' => 'web/assets/ezplatform/{$name}/',
```

Comment:
I'd stick to `ezplatform` as well. This only affects packages of type `ezplatform`, does it ?